learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -342,19 +367,19 @@ Below the member panel, exists the list of all available node types:
 
 .. image:: img/visual_script33.png
 
-Once you're comfortable and familiar with them, the Ctrl-F (Command-F on
-Mac) is very useful for searching the list.
+
+Once you're comfortable and familiar with them, the Ctrl-F (Command-F on Mac) is very useful for searching the list.
```

Comment:
Don't assume a level of comfort, just state the facts.  "Ctrl-F (Command-F on MacOS) allows you to search the list."